packages/angular/src/date-time-picker/date-time-picker.component.ts

```diff
@@ -233,13 +234,15 @@ export class DateTimePickerComponent implements OnChanges, OnInit {
 
   ngOnInit() {
     if (!this.selected) {
-      this.selected = [this.dateTimeRanges[0].key];
+      this.selected = [null];
     }
   }
 
   formatCurrentRangeTitle() {
     const [rangeOrType] = this.selected;
-    if (rangeOrType === 'RELATIVE' || rangeOrType === 'ABSOLUTE') {
+    if (!rangeOrType) {
+      return '';
```

Comment:
We should probably add a `placeholder` input to return in this case (I'm assuming if you're passing `[null]` you don't want a specific selection)